storybook/stories/Layout/Button.tsx

```diff
@@ -1,24 +1,129 @@
 import React from "react";
 import { storiesOf } from "@storybook/react-native";
-import { DarkButton } from "../../../src/components/Layout/Buttons/DarkButton";
-import { View } from "react-native";
+import {
+  Button,
+  DangerButton,
+  DarkButton,
+  HelpButton,
+  SecondaryButton,
+  TransparentButton,
+} from "../../../src/components/Layout/Buttons";
+import { AppText } from "../../../src/components/Layout/AppText";
+import { View, StyleSheet } from "react-native";
 import { size, color } from "../../../src/common/styles";
-import { Feather } from "@expo/vector-icons";
+import { Feather, Ionicons } from "@expo/vector-icons";
 
-storiesOf("Layout", module).add("DarkButton", () => (
-  <View style={{ margin: size(3) }}>
-    <DarkButton text="Checkout" />
-    <DarkButton text="Checkout" isLoading={true} />
-    <DarkButton
-      text="Checkout"
-      icon={
-        <Feather name="shopping-cart" size={size(2)} color={color("grey", 0)} />
-      }
-    />
-    <DarkButton
-      text="Scan"
-      icon={<Feather name="maximize" size={size(2)} color={color("grey", 0)} />}
-    />
-    <DarkButton text="Next customer" fullWidth={true} />
-  </View>
-));
+const styles = StyleSheet.create({
+  wrapper: {
+    margin: size(3),
+  },
+  item: {
+    marginBottom: size(1),
+  },
+});
+
+const darkButtonElements: JSX.Element[] = [
+  <DarkButton key="0" text="Checkout" />,
+  <DarkButton key="1" text="Checkout" isLoading={true} />,
+  <DarkButton
+    key="2"
+    text="Checkout"
+    icon={
+      <Feather name="shopping-cart" size={size(2)} color={color("grey", 0)} />
+    }
+  />,
+  <DarkButton
+    key="3"
+    text="Scan"
+    icon={<Feather name="maximize" size={size(2)} color={color("grey", 0)} />}
+  />,
+  <DarkButton key="4" text="Next customer" fullWidth={true} />,
+];
+
+const dangerButtonElements: JSX.Element[] = [
+  <DangerButton key="0" text="Danger" />,
+  <DangerButton key="1" text="Danger" isLoading={true} />,
+  <DangerButton key="2" text="Danger" fullWidth={true} />,
+  <DangerButton key="3" text="" accessibilityLabel="test button" />,
+];
+
+const secondaryButtonElements: JSX.Element[] = [
+  <SecondaryButton key="0" text="Default Secondary Button" />,
+  <SecondaryButton key="1" text="Secondary Button" isLoading={true} />,
+  <SecondaryButton key="2" text="Button disabled" disabled={true} />,
+  <SecondaryButton key="3" text="Small Button" size="small" />,
+  <SecondaryButton key="4" text="Medium Button" size="medium" />,
+  <SecondaryButton
+    key="5"
+    text="Icon Button"
+    icon={
+      <Feather name="shopping-cart" size={size(2)} color={color("grey", 100)} />
+    }
+  />,
+  <SecondaryButton key="6" text="FullWidth Button" fullWidth={true} />,
+];
+
+storiesOf("Layout/Button", module)
+  .add("Button", () => (
+    <View style={styles.wrapper}>
+      <Button text="Button" />
```

Comment:
Why does the default `Button` not have the alternative `sizes`, `props`, or other `text` ya?

Reply:
that's y its a default button (without props). what else do we need 🤔

Reply:
Oh as in, why not show the alternatives of the default Button as well? Unless the purpose of the storybook is to only show components that are used in the application? 🤔

Reply:
so far only customised application's used components. for Defaults let it be like that.